tests/integration/test_schema.py

```diff
@@ -1526,6 +1526,41 @@ async def test_schema_same_subject(registry_async_client: Client, trail: str) ->
     assert json == {"id": schema_id, "subject": subject, "schema": ujson.loads(schema_str), "version": 1}
 
 
+async def test_schema_same_subject_unnamed(registry_async_client: Client) -> None:
+    """
+    The same schema JSON should be returned when checking the same schema str against the same subject
```

Comment:
This could be a bit better, e.g.:

```
Avro equality doesn't depend on the value of `name`.
```

Reply:
There seems to be implicit assumption about "schema" is "avro schema" unless otherwise mentioned. E.g. filename is test_schema, as avro schemas where first, and only later jsonschema and protobuf was added. Maybe some bigger review of these could be done later?